models/layerzero/bnb/layerzero_bnb_send.sql

```diff
@@ -0,0 +1,211 @@
+{{ config(
+    schema = 'layerzero_bnb',
```

Comment:
If we defined schema in `dbt_project.yml`, we may remove this line, as it automatically inherit the schema.
```suggestion
```